ios/RNPurchases.h

```diff
@@ -3,12 +3,8 @@
 //  Copyright © 2019 RevenueCat. All rights reserved.
 //
 
-#if __has_include("RCTEventEmitter.h")
-#import "RCTEventEmitter.h"
-#else
+#import <React/RCTAppState.h>
```

Comment:
Why this new import?

Reply:
`RCTBridgeModule` is declared in that module, so the `RNPurchases` declaration falls on build without it.

Reply:
Hmm, I tried to build a new expo 44 app pointing to https://github.com/RevenueCat/react-native-purchases/pull/316/ and it worked fine without this new import. Also:

`React/RCTEventEmitter.h` is doing:

```
#import <React/RCTBridge.h>
```

And `RCTBridge.h` does:

```
#import <React/RCTBridgeModule.h>
```

So `RCTBridgeModule.h` shouldn't give a build error